recipe/const_auto_in_jit.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/torch/csrc/jit/python/init.cpp b/torch/csrc/jit/python/init.cpp
+index 23aa92f222..ce19ce20ec 100644
+--- a/torch/csrc/jit/python/init.cpp
++++ b/torch/csrc/jit/python/init.cpp
+@@ -396,7 +396,7 @@ void initJITBindings(PyObject* module) {
+             ArgumentSpecCreator arg_spec_creator(*graph);
+             Stack stack;
+             stack.reserve(inputs.size()); // captures?
+-            for (auto& obj : inputs) {
++            for (const auto& obj : inputs) {
+               stack.push_back(toTypeInferredIValue(obj));
+             }
+             ArgumentSpec spec = arg_spec_creator.create(with_grad, stack);
```

Comment:
Seems like there is another one on line 420. How does this work on CPython?

Reply:
honestly, i don't know. i fixed one so it would verify my hypothesis that this was the cause